internal/provider/resource_appdata_dsource.go

```diff
@@ -16,9 +16,9 @@ func resourceAppdataDsource() *schema.Resource {
 		Description: "Resource for appdata dSource creation.",
 
 		CreateContext: resourceAppdataDsourceCreate,
-		ReadContext:   resourceAppdataDsourceRead,
-		UpdateContext: resourceAppdataDsourceUpdate,
-		DeleteContext: resourceAppdataDsourceDelete,
+		ReadContext:   resourceDsourceRead,
```

Comment:
Does it mean only the create api is different for dsource creation? I mean in appdata or oracle we get the same outputs from the api?

Reply:
yes the output is same for both , only the inputs/create API  are different